react/features/conference/components/native/Conference.js

```diff
@@ -499,11 +523,21 @@ class Conference extends AbstractConference<Props, State> {
 function _mapStateToProps(state) {
     const { isOpen } = state['features/participants-pane'];
     const { aspectRatio, reducedUI } = state['features/base/responsive-ui'];
+    const { backgroundColor, backgroundImageUrl } = state['features/dynamic-branding'];
     const participantCount = getParticipantCount(state);
+    const brandingStyles = {
+        conference: {
+            alignSelf: 'stretch',
+            backgroundColor,
```

Comment:
What if the color is undefined?

Reply:
I tested that case and it doesn't break. I can add a check for that.